ark-cli/src/commands/storage/mod.rs

```diff
@@ -2,6 +2,8 @@ use clap::Subcommand;
 
 mod list;
 
+// FIXME: We should use new `fs-storage` crate to handle storage operations
```

Comment:
Currently, `ark-cli` has its own storage module, defined [here](https://github.com/ARK-Builders/ark-rust/blob/main/ark-cli/src/models/storage.rs). This implementation handles two types of storage (files and folders) and includes methods such as `append`, `read`, and `insert`. It is primarily used to store information about tags, scores, stats, properties, metadata, previews, and thumbnails.

I think this functionality should be moved to a separate crate. We could either create a new crate or include it as a module in `fs-storage`.

This isn't a significant amount of work, but it's necessary, just like having a crate for backup and restore functionality is essential. `ark-cli` should focus on defining CLI structs only.

Reply:
I just noticed that #69 would likely provide all the functionality we need for `ark-cli storage` and in a generic way

Reply:
@tareknaser maybe, but #69 is a bit abstract, I think we need to remove current storage imitation from `ark-cli` and replace it with methods from `fs-storage`.

Can be done in this PR if it's small amount of work, otherwise create new issue for it please and remove the FIXME.